Python/Product/NetStandard.settings

```diff
@@ -0,0 +1,48 @@
+<Project>
+  <Target Name = "ValidateAssemblyName" >
+    <Error Condition = " '$(AssemblyName)'=='' " Text = "AssemblyName property must be specified before R.Settings.NetCore.props is referenced" />
+  </Target>
+
+  <PropertyGroup>
+    <BuildRoot Condition="'$(BuildRoot)' == ''">$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), build.root))</BuildRoot>
+    <BuildOutputRoot Condition="'$(BuildOutputRoot)' == ''">$(BUILD_BINARIESDIRECTORY)</BuildOutputRoot>
+    <BuildOutputRoot Condition="'$(BuildOutputRoot)' == ''">$(BuildRoot)\BuildOutput\VsCode\</BuildOutputRoot>
+
+    <ObjDirectory Condition="'$(ObjDirectory)' == ''">$(BuildOutputRoot)obj\</ObjDirectory>
+    <BinDirectory Condition="'$(BinDirectory)' == ''">$(BuildOutputRoot)bin\</BinDirectory>
+
+    <BaseIntermediateOutputPath>$(ObjDirectory)\$(AssemblyName)\</BaseIntermediateOutputPath>
+    <IntermediateOutputPath>$(BaseIntermediateOutputPath)\$(Configuration)\</IntermediateOutputPath>
+
+    <BaseOutputPath>$(BinDirectory)</BaseOutputPath>
+    <OutputPath>$(BaseOutputPath)\$(Configuration)\</OutputPath>
+
+    <BuildDependsOn>ValidateAssemblyName;$(BuildDependsOn)</BuildDependsOn>
+    <AppendTargetFrameworkToOutputPath>false</AppendTargetFrameworkToOutputPath>
+  </PropertyGroup>
+
+  <PropertyGroup>
+    <ErrorReport>prompt</ErrorReport>
+    <WarningLevel>4</WarningLevel>
+    <DebugType>portable</DebugType>
+    <NetStandard>true</NetStandard>
+    <DefineConstants>NETSTANDARD;$(DefineConstants)</DefineConstants>
+  </PropertyGroup>
+
+  <!-- Signing -->
+  <PropertyGroup>
+    <MicroBuild_DoNotStrongNameSign>true</MicroBuild_DoNotStrongNameSign>
```

Comment:
Why do we need this if we're (a) strong name signing and (b) only strong name signing when building for release?

Reply:
It is for .NET Core only. We need to tests this but we had to add in RTVS since MB auto-signs in .NET Core projects by default